bedrock/products/templates/products/vpn/platforms/linux.html

```diff
@@ -0,0 +1,168 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import vpn_nav_platform_cta, vpn_platform_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% set vpn_countries = 30 %}
+{% set vpn_devices = 5 %}
+{% set vpn_devices_minus_1 = 4 %}
```

Comment:
The section of copy that uses this variable is:

> Take an extra measure of security when you’re online — both at home and out and about — with Mozilla VPN. For a small monthly fee, it uses advanced WireGuard® protocol to encrypt your internet connection on your Linux-based computer and **four other devices**, letting you stream shows, play games, shop, and go about your daily life online knowing that you’re secure.

I have a comment with the copywriting team about making this the maximum number. It also occurs in the iOS page, and needs fixing.